source/includes/odata/odata_basics.md

```diff
@@ -271,6 +271,8 @@ The server will ignore `$skip` results and then return the first `$top` items.
 
 When paging is applied, a link to the next results will be added at the end of the payload.
 
+Please note that the OData API does not allow a result window larger than 10000.
```

Comment:
I think we could be more explicit: 

```Please note that the OData API does not allow the resultset to be larger than 10,000 results.  So, for instance, you won't be able to request more than 100 pages of 100 results. To retrieve the whole content of datasets larger than 10000 records, you should use the OpenDataSoft Download API.```

Reply:
OK on the principle, but it's a bit weird for OData semantics though: if I ask the server to give me the $top 1000 results after $skipping 1000000, I expect the resultset size to be of 1000